types/omggif/index.d.ts

```diff
@@ -50,10 +50,13 @@ export class GifWriter {
 }
 
 export class GifReader {
+    height: number;
+    width: number;
+
     constructor(buf: Buffer);
 
-    decodeAndBlitFrameBGRA(frame_num: number, pixels: number[]): void;
-    decodeAndBlitFrameRGBA(frame_num: number, pixels: number[]): void;
+    decodeAndBlitFrameBGRA(frame_num: number, pixels: number[] | Uint8ClampedArray): void;
+    decodeAndBlitFrameRGBA(frame_num: number, pixels: number[] | Uint8ClampedArray): void;
```

Comment:
Why are you using `Uint8ClampedArray`? Why not just `Uint8Array`?
```suggestion
    decodeAndBlitFrameBGRA(frame_num: number, pixels: number[] | Uint8Array): void;
    decodeAndBlitFrameRGBA(frame_num: number, pixels: number[] | Uint8Array): void;
```

Reply:
`Uint8ClampedArray` is used because canvas.context2d.getImageData() returns value `{ data: Uint8ClampedArray }`, moreover `Uint8Array`  and `Uint8ClampedArray` and number[] are not interchangeable 🤔. So I think we could append both of them `Uint8Array` and `Uint8ClampedArray`.

Reply:
Okay, so you mean this?

```suggestion
    decodeAndBlitFrameBGRA(frame_num: number, pixels: number[] | Uint8Array | Uint8ClampedArray): void;
    decodeAndBlitFrameRGBA(frame_num: number, pixels: number[] | Uint8Array | Uint8ClampedArray): void;
```